CHANGELOG.md

```diff
@@ -79,6 +80,18 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+## [v0.65.0]
+
+> Release date: 2025/03/27
+
+- Added a Client API (GetById) for the following credentials: [#526](https://github.com/Kong/go-kong/pull/526)
```

Comment:
The link looked a little odd when added after the last element in list, so added in this line. Please let me know if I should move it to the end of list.